packages/@lightningjs/ui-components/src/mixins/withLayout/index.js

```diff
@@ -102,7 +102,7 @@ export default function withLayout(Base) {
             : calculatedWidth; // Width must be set first in order for Cards to be displayed properly
         this.h = calculatedHeight;
         if (this._itemLayout && this._itemLayout.circle && this.style) {
-          if (-1 < this._componentStyleSourceProps.indexOf('radius')) {
+          if (-1 < this.style?.radius) {
```

Comment:
what's the -1 for in this context? For the above, I think it makes sense as its checking if it exist

Reply:
Fixed